packages/frontend/component/src/ui/masonry/masonry.tsx

```diff
@@ -127,14 +178,17 @@ export const Masonry = ({
       const handler = throttle((e: Event) => {
         const scrollY = (e.target as HTMLElement).scrollTop;
         updateSleepMap(layoutMap, scrollY);
+        if (stickyGroupHeader) {
```

Comment:
The handler should run on mount to make sure one of the header is applied with sticky

Reply:
There is a delay when the first header get being rendered as sticky group. Maybe always render the container to avoid it?